components/switch/overview.md

```diff
@@ -37,116 +37,78 @@ The <a href = "https://www.telerik.com/blazor-ui/switch" target="_blank">Blazor
 
-This section shows the following two examples
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
 
-* [Remove Labels](#remove-labels)
-* [Custom Labels and Size](#custom-labels-and-size)
-
-
-### Remove Labels
-
->caption Switch without labels
+| Method | Description |
+| --- | --- |
+| `FocusAsync ` | programmatically focuses the Switch 
 
 ````CSHTML
-@* Use a blank space to remove visible labels *@
-
-<TelerikSwitch @bind-Value="@isSelected" OnLabel=" " OffLabel=" " />
+@* Use the Switch reference to programmatically focus the component *@
 
-@code {
-    private bool isSelected { get; set; }
-}
-````
+<TelerikButton OnClick="@FocusSwitch">Focus Switch</TelerikButton>
 
-### Custom Labels and Size
-
-The example below shows how to:
-
-* Bind the switch to a value
-* Set custom labels for its On and Off states
-* Use the `Width` parameter to control the size of the component
-
->caption Example that showcases the "I agree to the terms and conditions" basic scenario
-
-````CSHTML
-@* Use the OnLabel and OffLabel to customize the labels of the Switch. Use the Width parameter to manipulate the width of the component. *@ 
-
-<h2 class="text-info">Terms and conditions</h2>
-
-@if (hasAgreed)
-{
-    <div class="alert alert-success w-50">
-        Thank you for agreeing to our terms and conditions!
-    </div>
-}
-else
-{
-    <p class="w-50 text-justify">
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris mi lectus, ultrices sed libero et, tempor rutrum mauris. Praesent sit amet suscipit leo, ut hendrerit lacus. Mauris posuere, mi in elementum pretium, sem elit maximus mauris, ac tempus turpis nunc sed orci. Nunc velit lacus, rutrum et dui mattis, condimentum fermentum velit. Pellentesque et elit rhoncus, sodales nibh ac, faucibus tellus. Vestibulum vitae tempor tellus. Sed maximus sem quis est posuere, efficitur porttitor augue tincidunt. Sed viverra dapibus ullamcorper. Vestibulum ex arcu, molestie sed quam vulputate, aliquet cursus lectus. Aenean sollicitudin condimentum fringilla. Integer arcu justo, sollicitudin ut libero ut, posuere finibus sapien. Suspendisse hendrerit convallis urna.
-        Donec eu sodales dui, et consequat massa. Integer vitae euismod dui, id rhoncus tellus. Ut luctus leo eget sapien eleifend facilisis. Duis sed maximus tortor. Ut nunc nibh, pulvinar a enim eget, mattis sagittis sem. Mauris odio nibh, aliquet a erat sit amet.
-    </p>
-}
-
-<TelerikSwitch @bind-Value="@hasAgreed"
-               Id="mySwitch"
-               OnLabel="@myOnLabel"
-               OffLabel="@myOffLabel"
-               Width="100px">
-</TelerikSwitch>
-I agree to the terms and conditions
+<TelerikSwitch @bind-Value="@toggleSwitch" @ref="@TheSwitchRef" />
```

Comment:
Should be `SwitchRef` - see the Docs & Demos Style Guide in the blazor repo wiki